content/influxdb/v2.0/reference/flux/stdlib/pagerduty/endpoint.md

```diff
@@ -37,12 +37,12 @@ _**Data type:** String_
 The output function requires a `mapFn` parameter.
 
 ### mapFn
-A function that builds the record used to generate the POST request.
+({{< req >}}) A function that builds the record used to generate the POST request.
 Requires an `r` parameter.
 
 _**Data type:** Function_
 
-The returned record must include the following fields:
+The function must return a record that includes the following fields:
```

Comment:
`mapFn` accepts a table row (`r`) and returns a record that must include the following fields: